datareduction/datareducer.cpp

```diff
@@ -871,17 +904,17 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
                std::array<int32_t,3>& gridSize = technicalGrid.getLocalSize();
                std::vector<double> retval(gridSize[0]*gridSize[1]*gridSize[2]);
 
-               // Iterate through fsgrid cells and extract boundary flag
```

Comment:
Also, we have the xdx, ydy, zdz terms for all other fields, should we also thus include fg_dbxdx, fg_dbydy, fg_dbzdz?

Reply:
So these are not computed anywhere. I suppose we could write a DRO that would compute them at IO, instead of having them computed in the normal `calculateDerivatives()` every step or through some contraption checking whether IO is coming up, and storing them an cluttering more some fsgrid?

Reply:
As discussed and summarised here too; now we only compute and write the background volume derivatives, to go along with the perturbed volume derivatives' full jacobian output. more later and/or as part of the GPU rehaul.